tools/serverpod_cli/bin/util/command_line_tools.dart

```diff
@@ -45,6 +45,12 @@ class CommandLineTools {
 
   static Future<void> createTables(Directory dir, String name) async {
     var serverPath = p.join(dir.path, '${name}_server');
+    var _isNetstatAvailable = await existsCommand('netstat');
+    if (!_isNetstatAvailable) {
+      printWarning(
+          'netstat is not available on this platform. Please install it and tr running setup-tables manually from $serverPath.');
```

Comment:
Please check spelling. Also, "not available on this platform" implies that it's not possible to install, so the error message needs better wording.

Also, is this tested with both netstat installed and not installed?

Reply:
I did it manually on the PC and will check that on non installed part. Anyhow cli tests are passing `existsCommand('xxxxx')` function. So considering that will work with non-installed platforms too. I will check with the script on installed part and update you in few minutes.

Reply:
`netstat is not installed on this machine. Aborting tables creation. Please install it and run setup-tables script manually for tables creation.`

Does this sound good?

Yes will update the status on non-installed part soon in minutes

Reply:
### The machine which doesn't have `netstat` 

![image](https://user-images.githubusercontent.com/35523357/187968586-c212cc7b-2266-456c-aabc-fb48b6eb9289.png)

### The machine which has `netstat` 

![image](https://user-images.githubusercontent.com/35523357/187968841-86f27a2b-dde7-4129-b2c3-42c5dcdf4262.png)